spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/KubernetesConfigDataLocationResolver.java

```diff
@@ -120,11 +120,6 @@ protected abstract void registerBeans(ConfigDataLocationResolverContext resolver
 			ConfigDataLocation location, Profiles profiles, PropertyHolder propertyHolder,
 			KubernetesNamespaceProvider namespaceProvider);
 
-	protected final boolean isRetryEnabled(ConfigMapConfigProperties configMapProperties,
```

Comment:
we do not need this anymore, and because it is `protected`, imho: safe to be deleted.